mkdocs.yml

```diff
@@ -102,6 +102,7 @@ nav:
             - Back Office tabs:
                 - Back Office tabs:  administration/back_office/back_office_tabs/back_office_tabs.md
                 - Create dashboard tab: administration/back_office/back_office_tabs/create_dashboard_tab.md
+            - Add tab switcher in Content Type: administration/back_office/content_tab_switcher.md
```

Comment:
Did you consider placing it under 'Back Office elements'?

Reply:
yes, but I decided to put it right under the back office as it's not a tab, rather a switcher.